MaveDB.pm

```diff
@@ -264,16 +264,30 @@ sub run {
 
   my $all_results = {};
   foreach (@data) {
+
+    my $matches = get_matched_variant_alleles(
+      {
+        ref    => $vf->ref_allele_string,
+        alts   => $allele,
+        pos    => $vf->{start},
+        strand => $vf->strand
+      },
+      {
+       ref  => $_->{ref},
+       alts => [$_->{alt}],
+       pos  => $_->{start},
+      }
+    );
+
     # Check if scores are associated with single aminoacid changes
-    my $hgvsp = $_->{result}->{MaveDB_pro};
+    my $hgvsp = $_->{result}->{MaveDB_hgvs};
     if ($self->{single_aa_changes}) {
       next if defined $hgvsp && !_is_single_aa_change($hgvsp);
```

Comment:
```suggestion
      next if defined $hgvsp && $hgvsp =~ /:p\./ && !_is_single_aa_change($hgvsp);
```

As `hgvs` can be HGVSg as well, as mentioned above, we should actively check if HGVSp is actually a HGVSp and then do the AA matching. Otherwise lots of HGVSg based results are getting filtered out; e.g. - `1 43349266 43349266 G T`